### PR TITLE
Doc pages update to EE 8

### PR DESCRIPTION
**Pull request by jruzzi**
The changes in this branch update the introductory HTML files.

### CODE REVIEW

File: ws/javaee8/docs/list.html [hunk 1]
```diff
@@ -97,7 +97,7 @@
 <tr><td colspan="2" class="insidehead">Batch Processing 1.0</td></tr>
 
 <tr>
-<td class="grey1" height="19" width="22%"><a href="../batch/joboperator-api/docs/index.html">The JobOperator API Batch Sample Application</a><br></td>
+<td class="grey1" height="19" width="22%"><a href="mailto:glassfish@javaee.groups.io">The JobOperator API Batch Sample Application</a><br></td>
```
Comment (yaminikb) [inline ws/javaee8/docs/list.html]: Why did you have to change this? The relative path to the index.html is right.

Comment (jruzzi) [reply ws/javaee8/docs/list.html]: I must have accidentally changed it. I'll change it back

File: ws/javaee8/docs/UserREADME.html [hunk 2]
```diff
@@ -82,17 +85,17 @@
...
+    <b>Download and install JDK version 1.8.0_144, if necessary.</b>
+    If JDK version 1.8.0+ is not already on your system, you must 
     download and install it. The download is available at the 
     <a href="http://www.oracle.com/technetwork/java/javase/downloads/index.html">
     Java SE Downloads</a> page.
   </li>
   <li>
-    <b>Download and install Apache Maven version 3.0.5, if necessary.</b>
+    <b>Download and install Apache Maven version 3.3+, if necessary.</b>
 	If Apache Maven version 3.0.5 is not already installed on your system,
```
Comment (yaminikb) [inline ws/javaee8/docs/UserREADME.html]: 3.0.5 -> 3.3+

**Comment by jruzzi** [comment]
I updated the last Maven reference. The issues causing GitHub to think DeveloperREADME is a binary file was fixed on the last commit from September 7: https://github.com/javaee/glassfish-samples/pull/60/commits/6742430ea69403465fa2461bc05d1a064b27c9ff

let me know if you have any other changes.

**Comment by jruzzi** [comment]
Not sure if this is a bug in GitHub. Can you Merge the pull request and we can see if it merges it in as a binary? Like I said, after I fixed the garbage mark up, it committed the html and did not show binary when I pushed it out.

**Pull request merged**